### PR TITLE
PERP-87: draft api doc update for perpetual position draft

### PR DESCRIPTION
**Pull request by ji-cao**

### CODE REVIEW

File: src/trading-api/next/draft/bullish-trading-api.yml [hunk 1]
```diff
@@ -0,0 +1,6049 @@
...
+            - $ref: '#/components/schemas/UsdValue'
+        fundingPnl:
+          description: Funding PnL
+          allOf:
+            - $ref: '#/components/schemas/UsdValue'
+        markPriceAtSettlement:
+          description: Mark Price
+          allOf:
+            - $ref: '#/components/schemas/UsdValue'
+        indexPriceAtSettlement:
```
Comment (dexterhuang-blockone) [inline src/trading-api/next/draft/bullish-trading-api.yml:6005]: I think we should standardize this naming convention. We have `atSettlement` and `settledAt`. I prefer if we do `settlementIndexPrice`

File: src/trading-api/next/draft/bullish-trading-api.yml [hunk 2]
```diff
@@ -0,0 +1,6049 @@
...
+          description: Realized PnL
+          allOf:
+            - $ref: '#/components/schemas/UsdValue'
+        fundingRateAtSettlement:
+          description: Funding Rate
+          type: string
+          example: "10.0"
+        settledAtDatetime:
```
Comment (dexterhuang-blockone) [inline src/trading-api/next/draft/bullish-trading-api.yml:6013]: `settlementDatetime`

**Review by dexterhuang-blockone — COMMENTED** [review]
Let's hide our internal JIRA info form public repos.

File: src/trading-api/next/draft/bullish-trading-api.yml [hunk 3]
```diff
@@ -0,0 +1,6049 @@
...
+            $ref: '#/components/schemas/TradingAccountId'
+          required: true
+        - in: query
+          name: symbol
+          example: BTC-USD-PERP
+          schema:
+            $ref: '#/components/schemas/MarketSymbol'
+          required: false
+        - in: query
+          name: settledAtDatetime[gte]
```
[the remark this answers is not in the document]
Comment (dexterhuang-blockone) [reply src/trading-api/next/draft/bullish-trading-api.yml:3165]: `settlementDatetime`

File: src/trading-api/next/draft/bullish-trading-api.yml (hunk 1 above)
Comment (dexterhuang-blockone) [reply src/trading-api/next/draft/bullish-trading-api.yml:6005]: So the fields should all be `settlement *`

File: src/trading-api/next/draft/bullish-trading-api.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (ji-cao) [reply src/trading-api/next/draft/bullish-trading-api.yml:3165]: > 
we do have something like settledAtDatetime[gte]
       ```
 - in: query
          name: createdAtDatetime[gte]
          description: start timestamp of window, ISO 8601 with millisecond as string
          schema:
            $ref: '#/components/schemas/DateTime'
          required: true
        - in: query
          name: createdAtDatetime[lte]
          description: end timestamp of window, ISO 8601 with millisecond as string
          schema:
            $ref: '#/components/schemas/DateTime'
          required: true
```

**Review by dexterhuang-blockone — APPROVED** [review]

**Comment by ji-cao** [comment]
> Let's hide our internal JIRA info form public repos.

will do next time

**Pull request merged**